spacy_llm/models/hf/dolly.py

```diff
@@ -34,9 +24,17 @@ def __call__(self, prompts: Iterable[str]) -> Iterable[str]:  # type: ignore[ove
             self._model(pr, **self._config_run)[0]["generated_text"] for pr in prompts
         ]
 
+    @property
+    def model_name(self) -> str:
+        return f"databricks/dolly-{self._variant}"
+
+    @staticmethod
+    def get_model_variants() -> Iterable[str]:
+        return ["v2-3b", "v2-7b", "v2-12b"]
+
```

Comment:
I like the idea here and the fact that we'd type this as a `Literal` in the `dolly_hf` function, but it feels to me that `model` or `model_name`, including the "dolly-" part, is much more intuitive to users, and readable in the config, than only specifying the variant suffix.

Reply:
So you'd keep the `variant` concept, but use the fully qualified name like `databricks/dolly-v2-3b`?
I'd like not to name this `model` because I expect there to be some confusion in instantiating a `Model` class with a `model` argument. Maybe `name`?

Reply:
I personally think this would be nicest:
```
name = "dolly-v2-3b"
```
It's sort of a consensus, dropping the "databricks" prefix but not the "dolly" part so it's still readable by itself. And then I'd still have the `Literal` which is nice.

Reply:
Maybe internally it can be `model_name`?

Reply:
Updated in https://github.com/explosion/spacy-llm/pull/176/commits/f78adfcf8f1cee6694253832b2f4892ce2cc0203.